docs/templates/Object Definition/Property Sets/Property Sets for Materials/README.md

```diff
@@ -0,0 +1,31 @@
+Property Sets for Materials
+===========================
+
+The concept template _Property Sets for Materials_ describes how a material can be related to a single or multiple property sets. A property set contains a single or multiple properties. The data types of an individual property are single value, enumerated value, bounded value, table value, reference value, list value, and combination of property occurrences.
```

Comment:
Do you thing it is necessary to emphasize "single" for property sets and properties or just use multiple/many?

Also a proposal:
```suggestion
The concept template _Property Sets for Materials_ describes how a material can be related to a single or multiple property sets. A property set contains a single or multiple properties. The data type of an individual property can be one of single, enumerated, value, table, reference or list value, and combination of property occurrences.
```
Not sure what you mean by "and combination of property occurrences". The data type of a property can be a combination of property occurrences as in a complex property?

Reply:
> and combination of property occurrences

I had exactly the same question. It's just taken from the existing templates. See https://github.com/buildingSMART/IFC4.3.x-development/issues/399